tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -0,0 +1,375 @@
+import json
+import os
+import socket
+import threading
+import traceback
+import urlparse
+import uuid
+
+from .base import (CallbackHandler,
+                   RefTestExecutor,
+                   RefTestImplementation,
+                   TestharnessExecutor,
+                   extra_timeout,
+                   strip_server)
+from .protocol import (BaseProtocolPart,
+                       TestharnessProtocolPart,
+                       Protocol,
+                       SelectorProtocolPart,
+                       ClickProtocolPart,
+                       SendKeysProtocolPart,
+                       TestDriverProtocolPart)
+from ..testrunner import Stop
+
+import webdriver as client
+
+here = os.path.join(os.path.split(__file__)[0])
+
+webdriver = None
```

Comment:
Is this used anywhere? Only possible use seems to be in `_screenshot`, but if I know python variable scope correctly that's just a local variable. If it used anywhere, can you leave a comment here as a clue?